.github/workflows/build-fw.yml

```
@@ -50,10 +50,18 @@ jobs:
       - uses: actions/checkout@v4
         with:
           path: tt-zephyr-platforms
+          fetch-depth: 0
```

Rather than fetching all of git history, and then having a separate step for fetching tags, why not just fetch the needed commit with tags?

The suggestion just jumped out at me after looking here.
https://github.com/actions/checkout

```suggestion
          fetch-tags: true
```

---

Didn't know about this, applied